decidim-debates/spec/forms/decidim/debates/debate_form_spec.rb

```diff
@@ -65,6 +69,32 @@
     end
   end
 
+  context "when handling attachments" do
```

Comment:
Could you add a spec with what happens what if you upload a document that is not allowed? for example a ODT file ?